src/makeRobotAccountant.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return function getSum(x) {
+    return function(y) {
+      count++;
+
+      if (count <= 3) {
+        return x + y;
+      }
+
+      if (count > 3 && count % 2 !== 0) {
+        return x + y;
+      }
```

Comment:
in either case: return x + y;
can combine?

Reply:
Agree, you can combine these two if block